src/common/src/telemetry/mod.rs

```diff
@@ -95,21 +95,16 @@ struct Cpu {
 
 impl SystemData {
     pub fn new() -> Self {
-        let mut sys = System::new();
-
         let memory = {
             let total = system_memory_available_bytes();
             let used = total_memory_used_bytes();
             Memory { used, total }
         };
 
-        let os = {
-            sys.refresh_system();
```

Comment:
`System::refresh_system` has been removed

Reply:
Won't we need `refresh_all` or sth?

Reply:
`name()`, `kernel_version()` and `os_version()` are now associated methods while `refresh_all()` is an instance method, so I guess we don't need to call `refresh_all`.